dli_app/mod_wiki/controllers.py

```diff
@@ -159,3 +170,28 @@ def search():
     else:
         flash_form_errors(form)
         return render_template('wiki/home.html', form=form)
+
+@mod_wiki.route('/question', methods=['GET', 'POST'])
+@mod_wiki.route('/question/', methods=['GET', 'POST'])
+def question():
+    """ Email administrators"""
+    form = AskQuestionForm()
+    if form.validate_on_submit():
+        # Send email to administrators
+        users = [u.email for u in User.query.filter_by(is_admin=True)]
+        mail = Mail(current_app)
+        emailtitle = form.emailtitle.data
+        content = form.content.data
+        sender = 'cs490testing@gmail.com'
+        senderinfo = '\nPlease reply to the following email address: ' + form.email.data
```

Comment:
Remove the sender info and do this instead:
````Message(...params..., reply_to=form.email.data)````

Also, adding a bit more context about the message should help. Example:
"A new question was asked concerning the online DLI policies WikI!"